skimage/util/__init__.py

```diff
@@ -14,10 +14,16 @@
 from .unique import unique_rows
 from ._invert import invert
 from ._montage import montage
-from ._map_array import map_array
 from ._label import label_points
 
 
+# Temporary workaround
```

Comment:
Temporary workaround for what? Are you just testing a lazy load in skimage.util? Or is this working around a bigger issue. Is this a WIP? If this is intended to be merged, more context would be nice.

Reply:
OK, I updated the documentation string here to be more detailed.